contracts/modules/common/RelayerModule.sol

```diff
@@ -99,6 +99,9 @@ contract RelayerModule is BaseModule {
             // solium-disable-next-line security/no-call-value
             (success, returnData) = address(this).call(_data);
             refund(_wallet, startGas - gasleft(), _gasPrice, _gasLimit, requiredSignatures, msg.sender);
+        } else {
+            success = false;
```

Comment:
I believe this assignment is not necessary as the default value of `success` is already `false`.